test/fail_compilation/testhtml.html

```diff
@@ -1,6 +1,10 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
 <!--
 PERMUTE_ARGS: -d -dw
+TEST_OUTPUT:
+---
+Error: unrecognized file extension html
+---
```

Comment:
These html tests are pretty useless now, only need the one to verify that this error occurs.

Reply:
Then again, this error will happen if you try `dmd some.unknown_ext` too.